src/mcpd/exceptions.py

```
@@ -303,3 +323,57 @@ def __init__(self, message: str, operation: str = None, timeout: float = None):
+    Example:
+        >>> try:
+        >>>     result = client.call.time.get_current_time()
+        >>> except PipelineError as e:
+        >>>     print(f"Pipeline {e.pipeline_flow} failure: {e}")
+        >>>     if e.pipeline_flow == "response":
+        >>>         print("Tool was called but results cannot be delivered")
+        >>>     else:
+        >>>         print("Request was rejected by pipeline")
+    """
+
+    def __init__(
```

Ruff (0.14.10)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)